DockerfileJupyter

```diff
@@ -0,0 +1,42 @@
+#
+#	LEAN Jupyter Docker Container 20170802
+#	
+
+# Use base system for cleaning up wayward processes
+FROM quantconnect/lean:foundation
+
+MAINTAINER QuantConnect <contact@quantconnect.com>
+
+# Install Tini
+RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini && \
+    echo "1361527f39190a7338a0b434bd8c88ff7233ce7b9a4876f3315c22fce7eca1b0 *tini" | sha256sum -c - && \
+    mv tini /usr/local/bin/tini && \
+    chmod +x /usr/local/bin/tini
+
+#Install Jupyter
+RUN pip install jupyter
+	
+# Install Python.NET
+RUN apt-get update && apt-get install -y libglib2.0-dev \
+  && wget --quiet https://github.com/QuantConnect/pythonnet/archive/master.zip \
+  && unzip master.zip && cd pythonnet-master && python setup.py install
+
+# Setting some environment variables
+ENV WORK /root/Lean/Jupyter/bin/Release
+ENV LEAN /usr/lib/python2.7/Lean
+ENV PYTHONPATH=${LEAN}:${PYTHONPATH}
+
+# Copy Lean files to convinient locations
+COPY ./Jupyter/bin/Release ${LEAN}
+COPY ./Jupyter/*.ipynb ${WORK}/Examples/
+RUN rm -irf ${LEAN}/*Python*
+
+WORKDIR $WORK
+
+ENTRYPOINT [ "/usr/local/bin/tini", "--" ]
+CMD jupyter notebook --ip='*' --port=8888 --no-browser --allow-root
+
+# Usage: 
+# docker build -t quantconnect/lean:foundation -f DockerfileLeanFoundation .
+# docker build -t quantconnect/lean:jupyter -f DockerfileJupyter .
+# docker run -v "(absolute to your data folder):/root/Lean/Data" quantconnect/lean:jupyter 
```

Comment:
The projects in Examples won't work with this run command.  The problem is that the example projects live in the path `~/Lean/Jupyter/bin/Release/Examples` and the data folder is specified in the config as `../../../Data`. Therefore, when in one of the Example projects, the absolute path where Lean is looking for data is `~/Lean/Jupyter/Data` when this run command mounts it at `~/Lean/Data`. The data path should probably be configured to an absolute path in Lean